src/pages/Home.jsx

```diff
@@ -1,3 +1,36 @@
-const Home = () => <h1>Welcome to Voma</h1>;
+import styled from 'styled-components';
 
-export default Home;
+const StyledSection = styled.section`
+    width: 50%;
+    margin: 0 auto;
+    padding-top: 100px;
+    font-family: 'IBM Plex Sans', sans-serif;
```

Comment:
No need for this. We have a global style in place.

Reply:
removed the `font-family`, but didn't see the other styling in global style

Reply:
Here's a link to it: https://github.com/Code-For-Chicago/Voma-frontend/blob/75c072507e8e223bb83e8bb5fad9e801e0860d62/src/lib/GlobalStyles.js#L60